build.py

```diff
@@ -602,7 +607,16 @@ def create_dockerfile_buildbase(ddir, dockerfile_name, argmap):
 COPY . .
 ENTRYPOINT []
 '''
-        df += install_dcgm_libraries(argmap['DCGM_VERSION'])
+        if FLAGS.enable_gpu:
+            df += install_dcgm_libraries(argmap['DCGM_VERSION'])
+        else:
+            df += '''
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+            git \
```

Comment:
Isnt git installed earlier?
Also why is numa installed only for non-GPU build?

Reply:
numa is a required dependency. Since numa_utils is included everywhere. I can remove the dependency in a separate PR.

https://jirasw.nvidia.com/browse/DLIS-2749

Reply:
NUMA is a generic optimization

Reply:
@jbkyang-nvi why are is the numa install only for when --enable-gpu if off?

Reply:
It's because CUDA image installs NUMA. In can make the check on if it's UBUNTU 20.04

Reply:
It is confusing why git and numa are installed only for non-gpu. Can we just install them unconditionally with other deps. If they are already installed then apt-get will just skip them.